doc/manual/manual.tex

```diff
@@ -8485,6 +8488,231 @@ \subsubsection{The van Keken thermochemical composition benchmark}
+are nearly indistinguishable.
+
+Upon examining the solution at the final time, we note that the general structure of the 
+solution shown in Figure~\ref{fig:VOF_van_Keken-02} matches the form and the general structure 
+found in other versions of this benchmark such as the fourth image on the right in 
+Figure~\ref{fig:vk-1}.
+We also note that the differences in the shape of the interface based on a single refinement 
+as shown in Figure~\ref{fig:VOF_van_Keken-02} are minor, although still slightly visible.
+This is to be expected as refinement is a perturbation of the initial condition at a smaller 
+wave length.
+
+\begin{table}
+  \centering
+   \begin{tabular}{|c|c|c|c|}
+     \hline
+      Global Refinement & Number of Processors & VOF & DG \\
+      \hline
+\phantom{1}5 & \phantom{1}4 &\phantom{6}1.33 minutes\phantom{hours} &\phantom{1}2.57 minutes  \\
+\phantom{1}6 & \phantom{1}4 &\phantom{1}8.51 minutes\phantom{hours} & 19.5\phantom{7} minutes \\
+\phantom{1}7 & \phantom{1}4 &\phantom{1}1.15 hours\phantom{minutes} & \phantom{1}2.49 hours\phantom{tes} \\
+\phantom{1}8 & \phantom{1}4 &\phantom{1}8.53 hours\phantom{minutes} & 19.6 hours\phantom{tes} \\
+\phantom{1}9 & 16           &         16.30 hours\phantom{minutes}  & \phantom{1}2.72 days\phantom{tes} \\
+          10 & 16           & \phantom{1}5.17 days\phantom{minutess}&>6.00 days\phantom{tess} \\
+            \hline
+        \end{tabular}
+        \caption{\it Comparison of runtimes for the van Keken problem with VOF and a DG computational field, in which the initial conditions for DG smoothed are as described in section~\ref{paragraph:van-keken compositional fields} above.
+            The times shown are for the full computation, ending at $t_{end} = 2000$ with a CFL
+            number of $\sigma=0.5$ in both cases. 
+            All of these computations were made with ASPECT version 2.2.0-pre (master, ef542ecc2) in release mode on the Peloton2 cluster at U.C.~Davis.
+            We note that the change in the CFL number $\sigma$ and the differing
+            choice of cluster makes a direct quantitative comparison between this
+            table and table~\ref{tab:runtime-table} invalid due to too many
+            confounding factors.
+        }
+        \label{tab:vof-runtime-comparison-table}
+\end{table}
+
+The consistency of the results shown here differs noticeably from the behavior of the  
+problem with discontinuous initial conditions when computed with the FEM and DG advection 
+algorithms.
+One of the possible reasons for these differences is the procedure for 
+defining the initial interface.
+As described above, we have designed an approach for setting the initial condition that should 
+reproduce a smooth curve to a high degree of accuracy and, in particular, the curve, which is
+where the discontinuity occurs in the discontinuous initial condition version of this problem.
```

Comment:
@naliboff, @egpuckett  Would this edit work, or do you have any suggestions on how else to clarify?

```suggestion
As described above, we have designed an approach for setting the initial condition that should
reproduce the smooth curve for the initial boundary between the two fluids for this problem
to a high degree of accuracy.
```